app/containers/LanguageProvider/reducer.js

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * LanguageProvider reducer
+ *
+ */
+
+import { fromJS } from 'immutable';
+import {
+  CHANGE_LOCALE,
+} from './constants';
+
+const initialState = fromJS({
+  locale: 'en',
```

Comment:
@justingreenberg if you can make a PR for this as well, that would be a great help.